src/view/com/notifications/FeedItem.tsx

```diff
@@ -231,10 +231,9 @@ let FeedItem = ({
             />
             {authors.length > 1 ? (
               <>
-                <Text style={[pal.text, s.mr5, s.ml5]}>
-                  {' '}
-                  <Trans>and</Trans>{' '}
-                </Text>
+                <Trans>
+                  <Text style={[pal.text]}> and </Text>
+                </Trans>
```

Comment:
is this one necessary? it will throw off existing translations and force the translations to include the surrounding spaces, which we try to avoid